tests/installation/upgrade_select.pm

```diff
@@ -39,7 +39,10 @@ sub run {
         send_key $cmd{next};
     }
     if (match_has_tag("select-for-update")) {
-        send_key $cmd{next};
+        assert_screen('select-for-update-arch');
```

Comment:
Yes, here can be assert_screen("select-for-update-$arch"); to make it more specific for each platform.
$arch = get_var("ARCH");

Reply:
Thanks, I changed the code as your suggestion, please help to review.